client/README.md

```diff
@@ -2,8 +2,47 @@
 [![NPM Package](https://img.shields.io/npm/v/@stacks/blockchain-api-client.svg?style=flat-square)](https://www.npmjs.org/package/@stacks/blockchain-api-client)
 [![Build Status](https://github.com/blockstack/stacks-blockchain-api/workflows/Build/badge.svg)](https://github.com/blockstack/stacks-blockchain-api/actions)
 
-Client for the Stacks Blockchain API.
+A JS Client for the Stacks Blockchain API. This includes a JSON RPC WebSocket client to subscribe to real-time updates, with full type safety.
 
-### Known Issues
+![image](https://user-images.githubusercontent.com/1447546/89547223-b8aa0980-d7c2-11ea-9aea-658a9dc96a67.png)
```

Comment:
The code doesn't match these screenshots anymore

Reply:
I added a "features" section to replace what I wanted to communicate through the screenshots